### DISCUSSION TITLE
신규 웹사이트 제보

### DISCUSSION THREAD

**Discussion by gaeulbyul** [List-KR/community#47] [Ad-Shield]
- https://www.newdaily.co.kr/site/data/html/2023/11/16/2023111600277.html
- https://www.meconomynews.com/
- https://www.brandbrief.co.kr/

위 사이트에서 애드쉴드 스크립트를 발견했습니다.

**Comment by piquark6046** [comment]
뉴데일리는 애드쉴드 보호가 적용되어 있고, 다른 웹 사이트들은 애드쉴드 analytics가 적용되어 있는 것 같습니다.
맞나요?

**Reply by piquark6046** [reply]
https://github.com/AdguardTeam/AdguardFilters/commit/f9f75a485c7e47f1597d3d5e25e337e726eed1d1
AdguardFilters에는 반영했습니다.

**Comment by JobcenterTycoon** [comment]
Fixed Ad-Shield analytics and ads for uBO

https://github.com/uBlockOrigin/uAssets/commit/633ae42af3c8ce47c39210bedebc7ec9c8150467
https://github.com/uBlockOrigin/uAssets/commit/6e95c1a95b797d332b7caa5ac91fb3e1d98c5463

**Comment by gaeulbyul** [comment]
신규 웹사이트 제보합니다.

- https://thestockmarketwatch.com/ ( https://github.com/AdguardTeam/AdguardFilters/issues/167046 )
- https://capress.kr/
- https://carandmore.co.kr/
- https://maxmovie.kr/
- https://motorgraph.com/
- https://newsbell.co.kr/
- https://tminews.co.kr/
- https://thehousemagazine.kr/
- https://www.hardreset.info/

Most of these websites doesn't seems to have the "protected" ad, except thestockmarketwatch.com .

@JobcenterTycoon I found another websites that contains the Ad-Shield's script.

**Reply by JobcenterTycoon** [reply]
Fixed.

**Reply by piquark6046** [reply]
@JobcenterTycoon Please create an exception filter related to the following changes in EasyList:
https://github.com/easylist/easylist/commit/f54047db352ef4ca478e5271b3088239132da6d3

**Reply by JobcenterTycoon** [reply]
Done.

**Reply by piquark6046** [reply]
Fixed in AdguardFilters and List-KR filters list.

**Comment by gaeulbyul** [comment]
신규 웹사이트 제보합니다:
* https://www.maketecheasier.com/
* https://www.tweaksforgeeks.com/
* https://www.issuya.com/

아직 위 사이트에서 보호된 광고는 발견하지 못했습니다.

**Reply by JobcenterTycoon** [reply]
Ad reinsertion found on https://www.tweaksforgeeks.com/

**Reply by piquark6046** [reply]
The ad reinsertion is fixed.

**Comment by gaeulbyul** [comment]
신규 웹사이트 제보합니다.

* https://topstarnews.net/
* https://www.allthetests.com/
* https://www.motorbikecatalog.com/
* https://interfootball.heraldcorp.com/ (interfootball.co.kr와 동일 사이트로 보입니다)

**Reply by JobcenterTycoon** [reply]
All fixed on uBO now. `https://www.allthetests.com/` already using ad reinsertion but with no PHP part.

**Reply by piquark6046** [reply]
fixed.

**Comment by gaeulbyul** [comment]
* https://www.etnews.com/
* https://www.iusm.co.kr/

**Reply by piquark6046** [reply]
The domains will be added into microShield userscript. Not AdguardFilters.

**Reply by piquark6046** [reply]
https://github.com/List-KR/microShield/commit/a70b1457cb7200340219f0922de7497434f8e282

**Comment by Mynameiskernan** [comment]
https://meeco.kr/

**Reply by piquark6046** [reply]
이미 추가되어 있습니다.

**Reply by Mynameiskernan** [reply]
답변해주셔서 감사합니다.

그제부터 새로운 화면이 뜨면서 사이트 이용이 안되어서 문의드립니다.

Microshield를 설치해도 동일합니다.
![IMG_4797](https://github.com/List-KR/community/assets/162894495/e36ab70d-7742-4cb5-90fd-678ec376fc1b)

사용환경은 ios입니다

**Reply by piquark6046** [reply]
유저스크립트를 업데이트해보셨나요?

**Reply by piquark6046** [reply]
또, 어떤 애드블록을 사용하시나요?

**Reply by Mynameiskernan** [reply]
애드가드 사용중이고 micro shield는 지난 일요일에 처름으로 설치했습니다. 최신버전으로 설치했다고 생각합니다.

**Reply by piquark6046** [reply]
3월 8일 쯤에 추가했는데요. 혹시 모르니, Safari에 있는 '가가' 버튼 -> Userscripts 앱에서 유저스크립트 업데이트를 확인해보세요.

**Reply by Mynameiskernan** [reply]
check updates 버튼 눌러도 똑같은 증상 나타납니다..🥲

**Reply by piquark6046** [reply]
check updates 버튼을 누르시고 업데이트가 있다고 하면 다운로드 버튼을 각각 누르시고 새로고침하시면 됩니다.

**Reply by Mynameiskernan** [reply]
check update 버튼을 눌러도 새로운 업데이트가 뜨진 않습니다!

**Reply by piquark6046** [reply]
Safari에 있는 '가가' 버튼을 누르신 상태의 스크린샷을 보내주시겠어요?

**Reply by piquark6046** [reply]
안 보입니다.

**Reply by Mynameiskernan** [reply]
![IMG_4811](https://github.com/List-KR/community/assets/162894495/adddc013-fdac-4fe3-a0e8-3a4d2d53c462)

**Reply by piquark6046** [reply]
Userscript 앱 클릭해보세요. 또 스크린샷도 보내주세요.

**Reply by Mynameiskernan** [reply]
![IMG_4821](https://github.com/List-KR/community/assets/162894495/b50dff3d-6e53-4125-ab8d-a074e085850e)
![IMG_4820](https://github.com/List-KR/community/assets/162894495/b189fdee-0737-4377-a447-46eec402469b)

귀찮게 해드리는 것 같아 죄송합니다

**Reply by piquark6046** [reply]
구름과 화살표가 있는 아이콘을 클릭해보세요.

**Reply by Mynameiskernan** [reply]
https://github.com/List-KR/community/assets/162894495/459a4e44-caa4-4025-a427-1952756f1818

**Reply by piquark6046** [reply]
그럼, 그 상태에서 미코에 방문해보세요.

**Reply by piquark6046** [reply]
그래도 안된다면, Safari의 추적 보호 기능을 꺼보세요.

**Reply by Mynameiskernan** [reply]
![IMG_4841](https://github.com/List-KR/community/assets/162894495/3fdbab5a-a3fa-4f17-8984-d0629bd4288e)

혹시 여기 크로스 사이트 추적 보호 말씀이신가요?

꺼도 동일합니다

**Reply by piquark6046** [reply]
대답이 느리신 것 겉은데 GitHub에 익숙하지 않으신 것 같습니다.
이메일로 연락하시겠습니까? filters@adguard.com.

**Reply by Mynameiskernan** [reply]
답장이 늦어 죄송합니다 이메일 드리겠습니다
